worker/migrations/migrations.go

```diff
@@ -129,51 +123,46 @@ func migrateAccountsToOrganization(domain string) error {
 		Account string `json:"account"`
 		Slug    string `json:"konnector"`
 	}
+	var errm error
 	for _, t := range triggers {
 		if t.Infos().WorkerType != "konnector" {
 			continue
 		}
 		err := t.Infos().Message.Unmarshal(&msg)
-		if err == nil && msg.Account != "" && msg.Slug != "" {
-			manifest, err := app.GetKonnectorBySlug(inst, msg.Slug)
-			if err != nil {
-				return err
-			}
-			vLink, err := json.Marshal(manifest.VendorLink)
-			if err != nil {
-				return err
-			}
-			link := string(vLink)
-			// Remove quotes
-			if len(link) > 0 && link[0] == '"' {
-				link = link[1:]
-			}
-			if len(link) > 0 && link[len(link)-1] == '"' {
-				link = link[:len(link)-1]
-			}
-			acc := &account.Account{}
-			if err := couchdb.GetDoc(inst, consts.Accounts, msg.Account, acc); err != nil {
-				return nil
-			}
-			encryptedCreds := acc.Basic.EncryptedCredentials
-			login, password, err := account.DecryptCredentials(encryptedCreds)
-			if err != nil {
-				return err
-			}
-			cipher, err := createCipher(orgKey, msg.Slug, login, password, string(link))
-			if err != nil {
-				return err
-			}
-			if err := couchdb.CreateDoc(inst, cipher); err != nil {
-				return err
-			}
-			settings.UpdateRevisionDate(inst, setting)
+		if err != nil || msg.Account == "" || msg.Slug == "" {
+			continue
+		}
+		manifest, err := app.GetKonnectorBySlug(inst, msg.Slug)
+		if err != nil {
+			errm = multierror.Append(errm, err)
```

Comment:
I think we should also put a `continue`, as with an empty manifest, we will accumulate several other errors for this trigger.

Reply:
Right, it is also true when there is an error on the account doc retrieval

Reply:
Yes, and we should check the other places where multierror is used. At least, the `buildCipher` error should also have its `continue`.